src/consensus/consensus.h

```diff
@@ -26,12 +26,9 @@ static const int64_t MAX_BLOCK_SIGOPS_COST = 80000;
 static const int COINBASE_MATURITY = 100;
 
 /** Flags for nSequence and nLockTime locks */
-enum {
-    /* Interpret sequence numbers as relative lock-time constraints. */
-    LOCKTIME_VERIFY_SEQUENCE = (1 << 0),
-
-    /* Use GetMedianTimePast() instead of nTime for end point timestamp. */
-    LOCKTIME_MEDIAN_TIME_PAST = (1 << 1),
-};
```

Comment:
1. Sure, that case could be made. Any suggestion on a proper name for that enumeration?
2. Changing existing `const` to `constexpr` is not in the scope of this PR :-)
3. When introducing new compile time constants I prefer `constexpr` which is explicit about the intention of the code (getting a *compile-time* constant). Do you see any compelling reasons to avoid using `constexpr` for compile-time constants?